src/tree-manager/index.js

```diff
@@ -278,6 +275,19 @@ class TreeManager {
 
     return keyboardNavigation.handleToggleNavigationkey(action, prevFocus, readOnly, onToggleChecked, onToggleExpanded)
   }
+
+  _getAddOnMatch(matches, searchTerm) {
+    let isMatch = (node, term) => node.label.toLowerCase().indexOf(term) >= 0
+    if (this.searchPredicate && typeof this.searchPredicate === 'function') {
+      isMatch = this.searchPredicate
+    }
+
+    return node => {
+      if (isMatch(node, searchTerm) === true) {
```

Comment:
Redundant true comparison?

Reply:
In the custom function they could return anything they wanted that could be evaluated to true? Should we let them? I mean return "hello world" would we valid?

Reply:
We kinda don't care... Same concept as `array.filter` or any other native matcher. We just care that the value is `truthy` since the return value is not being used anywhere.